client/src/components/EmptyListScreen.js

```diff
@@ -0,0 +1,37 @@
+import styled from "@emotion/styled";
+import React from "react";
+import logoSrc from "../assets/logo.svg";
+
+function EmptyListScreen(props) {
```

Comment:
I would recommended destructuring assignment of the props. That way another dev can see what kind of props you use. In this case it seems like you don't use any props at all? Is that correct? If so, remove the "props" argument.
Also you might consider adding PropTypes to your components so other developers can easily understand what kind of props you use and automated testing can check the types to warn you in case of wrong prop types.

Reply:
Good point. I will not use them here and forgot to delete them. 👍